akka-actor/src/main/resources/reference.conf

```
@@ -1100,6 +1100,9 @@ akka {
     # This property is related to `akka.jvm-shutdown-hooks` above.
     run-by-jvm-shutdown-hook = on
 
+    # Run the coordinated shutdown when ActorSystem.terminate is called
+    run-by-actor-system-terminate = on
```

Seems nicer but I guess it's leveraging the existing `run-by` convention which reads oddly.
```suggestion
    actor-system-terminate-enabled = on
```

---

I tried to fit in with the existing property `run-by-jvm-shutdown-hook`, which is similar

we already have `terminate-actor-system`, which is kind of opposite direction, and that wouldn't be very clear with `actor-system-terminate-enabled`